traits/observers/_dict_change_event.py

```diff
@@ -21,9 +21,11 @@ class DictChangeEvent:
     "*name* _items" trait. In particular, the attribute ``changed`` is not
     defined here.
 
+    The interface of this object is provisional as of version 6.1.
```

Comment:
If we keep this sentence, I will open a separate issue for removing this again when the interface is no longer provisional.

Reply:
Opened #1110